### DISCUSSION TITLE
Install in windows xp?

### DISCUSSION THREAD

**Discussion by nitro912gr** [arkypita/LaserGRBL#1227] [Q&A] [answered]
Hello and thank you very much for this great software.

I wanted to install in windows XP but I can't figure out how.

I download the last install.exe and it can't run on XP, I then downloaded the zip below but I don't know what to do with the files in there.

Thanks :)

**Comment by StuartB4** [comment]
You may need to install MS .Net Framework for it to run on XP. 
From here: https://dotnet.microsoft.com/download/dotnet-framework/net48
The 4.8 Runtime one on the right.

**Comment by arkypita** [comment]
@nitro912gr sorry for that. I have recently updated the tool I use to build the installer, without realizing that in the meantime they had removed support for XP. 

![image](https://user-images.githubusercontent.com/8782035/103155475-af84fd80-47a0-11eb-8aff-f4983e1953b9.png)

I will try to go back for the installer of the next versions, meanwhile you can install the v3.8.0 version which is packaged with the old installer version: https://github.com/arkypita/LaserGRBL/releases/tag/v3.8.0

**Reply by nitro912gr** [reply]
Hello thanks for this but if there is no demand for the XP installer in general please don't bother just for me.

:)

**Reply by weshatheleopard** [reply]
There is. I have the old laptop I intend to use to drive the engraver, and it runs XP.

Last .NET available for Windows XP is 4.0, can be downloaded [here](https://www.microsoft.com/en-gb/download/details.aspx?id=17718) (direct link to EXE [here](https://download.microsoft.com/download/9/5/A/95A9616B-7A37-4AF6-BC36-D6EA96C8DAAE/dotNetFx40_Full_x86_x64.exe)), also available on [archive.org](https://web.archive.org/web/20240201202332/https://download.microsoft.com/download/9/5/A/95A9616B-7A37-4AF6-BC36-D6EA96C8DAAE/dotNetFx40_Full_x86_x64.exe). LaserGRBL 5.7.0 seems to run fine on it.

**Comment by arkypita** [comment, marked as answer]
The last version is XP compatible: https://github.com/arkypita/LaserGRBL/releases/tag/v3.8.7
Only 0.6% use WinXP, but it doesn't cost me anything, so why not.

**Reply by nitro912gr** [reply]
thank you so much, I just already have a netbook with win XP that operates a vinyl cutter and didn't want to add another system for operating my atomstack.
